nautobot/utilities/templates/utilities/templatetags/dynamic_filter_form_modal.html

```diff
@@ -0,0 +1,52 @@
+{% load form_helpers %}
+{% load static %}
+
+<div class="modal fade" tabindex="-1" id="{{ form_name }}_modal">
+    <div class="modal-dialog modal-lg" style="width: 90% !important;">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Filter Form</h4>
+            </div>
+            <div class="modal-body">
+                {% if form %}
+                    <form id="dynamic-filter-form" class="form-horizontal" method="get" action=".">
+                        <div class="">
+                            <table class="table" id="secrets" style="table-layout: fixed;">
```

Comment:
`table-layout: fixed` makes it so the table doesn't adjust columns to fit their contents, right? Is that why there's all that dead whitespace to the right of the "delete" button column? Not blocking for this PR but it'd be nice to fix that up with some smarter column widths.

Reply:
The white space to the right of the "delete" button results from the equal width of each column.